libcontainer/nsenter/nsexec.c

```diff
@@ -311,7 +311,7 @@ static int try_mapping_tool(const char *app, int pid, char *map, size_t map_len)
 
 static void update_uidmap(const char *path, int pid, char *map, size_t map_len)
 {
-	if (map == NULL || map_len <= 0)
+	if (map == NULL || !map_len)
```

Comment:
I would prefer `map_len == 0` as it's more readable and is in harmony with the `map == NULL` check.

Reply:
Thanks! Make sense to me.

Updated, PTAL! @kolyshkin